otter/test/convergence/test_gathering.py

```diff
@@ -69,14 +70,15 @@ def resolve_svcreq(eff, result, service_type,
     return resolve_effect(eff, result)
 
 
-def svc_request_args(changes_since, limit):
+def svc_request_args(**params):
     """
     Return service request args with formatted changes_since argument in it
     """
-    params = urlencode([('changes_since', changes_since.isoformat() + 'Z'),
-                        ('limit', limit)])
+    changes_since = params.get('changes-since', None)
```

Comment:
I'm not sure why you made this change, it just seems to make it harder to pass the parameter (i.e. how you have to use `**{'changes-since': ...}` instead of `change_since=...`.   IMO it'd be better to leave the parameter name the same but then just construct `params` appropriately inside this method.